helpers.go

```diff
@@ -200,10 +201,10 @@ func preventImageOverwrite(repo, tag string) error {
 	imageExists, err := checkImageExists(repo, tag)
 
 	if err != nil {
-		return fmt.Errorf("Cannot proceed safely: %v.", err)
+		return fmt.Errorf("cannot proceed safely: %v", err)
 	}
 	if imageExists {
-		return fmt.Errorf("Cannot overwrite an existing image. Please use a different repository/tag.")
+		return errors.New("cannot overwrite an existing image. Please use a different repository/tag")
```

Comment:
We are pretty inconsistent for this to be honest, but if we follow the documentation, `errors.New` is the proper function. That being said, I don't have any strong feelings, and I think that this change is unrelated to the PR anyways ...